modules/consensus/consensusset.go

```diff
@@ -14,14 +14,32 @@ import (
 	"github.com/NebulousLabs/Sia/types"
 )
 
+const (
+	// fullVerification indicates that a block should be fully verified when
+	// being loaded from disk.
+	fullVerification verificationRigor = 0
+
+	// partialVerification indicates that transaction verification can be
+	// skipped. Transaction verification is computationally intensive, and
+	// skipping such a step noticably increases speed when loading many blocks
+	// at once. Usually, partialVerification is used when loading blocks from
+	// disk.
+	partialVerification verificationRigor = 1
+)
+
+// verificationRigor is a type indicating the intensity of verification that
+// should be using while accepting a block. For blocks that come from trusted
+// sources, the computationally expensive steps can be skipped.
+type verificationRigor byte
```

Comment:
minor quibble, but I would use a `uint8` here. The types are equivalent, but one implies an integer and one implies a character. (ditto for `rune` and `int32`)